android/modules/ui/src/java/ti/modules/titanium/ui/widget/tabgroup/TiUIAbstractTabGroup.java

```diff
@@ -425,6 +454,47 @@ public void updateTitle(String title)
 		}
 	}
 
+	public Drawable updateIconTint(TiViewProxy tabProxy, Drawable drawable, boolean selected)
+	{
+		final KrollDict tabProperties = tabProxy.getProperties();
+		final KrollDict properties = getProxy().getProperties();
+
+		if (drawable == null) {
+			final String title = tabProperties.optString(TiC.PROPERTY_TITLE, "");
+			Log.w(TAG, "Could not update tab icon tint color for '" + title + "' tab.");
+			return null;
```

Comment:
Can you remove the "Could not update tab icon tint color..." warning log message please?
If I don't set any tab icons, then I end up seeing warning messages every time I select a tab.
Note that I think the top tab style looks fine without icons.